cmd/sso-auth/main.go

```diff
@@ -42,9 +44,12 @@ func main() {
 	}
 	defer authMux.Stop()
 
+	requestTimeout := config.ServerConfig.TimeoutConfig.Request
+	shutdownTimeout := requestTimeout + 1*time.Second
```

Comment:
I'm not sure there's any real value in having the shutdown timeout be a little longer than the request timeout we already apply to this server.  I don't think there's much harm in this, but 1 second is pretty arbitrary and I'd be happy to drop this and just use the request timeout as the shutdown timeout.